Auxiliary/DirectXTexXbox.h

```diff
@@ -163,16 +163,16 @@ namespace Xbox
 
     HRESULT __cdecl CreateTexture(
         _In_ ID3D12Device* d3dDevice,
-        _In_ const XboxImage& xbox, _Outptr_opt_ ID3D12Resource** ppResource, _Outptr_ void** grfxMemory);
+        _In_ const XboxImage& xbox, _Outptr_opt_ ID3D12Resource** ppResource, _Outptr_ const uint8_t** grfxMemory);
```

Comment:
Note ``void*`` is actually the correct annotation for graphics memory. This is untyped memory in the video device range so could be really anything and maybe not byte accessible.